.vsts-ci/releaseBuild.yml

```diff
@@ -17,7 +17,7 @@ resources:
 
 jobs:
 - job: build_windows
-  pool: Package ES Lab A
+  pool: Package ES CodeHub Lab E
```

Comment:
This should be the standard pool `Package ES Standard Build`. Did you have problems using it? It seems you had it in one of your previous commits

Reply:
Yes, the build didn't start when using `Package ES Standard Build`. It says something like "it doesn't exist or I don't have permission". I copied this one from Paul's SecretManagement release build.

Reply:
I just sent the email to request adding the agent pool to PSReadLine. I will update the pool name once the agent pool is added and my access is clear.
Before that, I would like to go with "Package ES CodeHub Lab E", as I need to release 2.2.0-beta1 today or tomorrow.